app/routes/signin-oidc.js

```diff
@@ -115,6 +115,21 @@ function getRedirectPath(
   }
 }
 
+function safelyGetLoginSource(request) {
+  try {
+    return JSON.parse(
+      Buffer.from(request.query.state, "base64").toString("ascii"),
+    ).source;
+  } catch (queryStateError) {
+    request.logger.setBindings({ query: request.query });
+    request.logger.setBindings({
+      query: request.query,
```

Comment:
duplicate entry

Reply:
You both spotted this, shame I didn;t :D